src/Neo/SmartContract/ApplicationEngine.cs

```diff
@@ -105,13 +109,22 @@ public partial class ApplicationEngine : ExecutionEngine
 
         /// <summary>
         /// GAS spent to execute.
+        /// In the unit of datoshi, 1 datoshi = 1e-8 GAS, 1 GAS = 1e8 datoshi
         /// </summary>
+        [Obsolete("This property is deprecated. Use FeeConsumed instead.")]
         public long GasConsumed { get; private set; } = 0;
```

Comment:
Please, create an issue to remove this deprecated code in some ~3.9.0 version, otherwise it'll hang here forever.